build_projects/shared-build-targets-utils/Utils/DotNetCli.cs

```diff
@@ -7,11 +7,9 @@
 
 namespace Microsoft.DotNet.Cli.Build
```

Comment:
Do you have an idea on how to address the source include issue? I don't want to source include dirs.